core/src/main/java/com/linecorp/armeria/client/circuitbreaker/NonBlockingCircuitBreaker.java

```diff
@@ -141,6 +146,16 @@ public boolean canRequest() {
         return true;
     }
 
+    @Override
+    public void enterState(CircuitState circuitState) {
```

Comment:
nit: rnn?

Reply:
didn't understand this comment 😅

Reply:
It should be **r**equire**N**on**N**ull. 🤣